arbitrator/arbutil/src/pricing.rs

```diff
@@ -6,8 +6,6 @@ use crate::{
     Bytes32,
 };
 
-use crate::evm::api::Ink;
```

Comment:
nit: why are we making changes to this folder?

Reply:
same as celestia